analysis.js

```diff
@@ -97,29 +108,36 @@ function calcDiceRolls(player) {
     } else
         countGame(data[game_id]);
 
+    dice_cnt.forEach(function(d, i) {
+        var y0 = 0;
+        d.total = d3.sum(d3.entries(d), function(d) {
+            return d.value;
+        })
+        d.bars = cur_players.map(function(name) {
+            return {
+                die:i+1,
+                name: name,
+                y0: y0,
+                y1: y0 += d[name],
+                d_perc: ((d[name] / d.total) * 100).toFixed(2),
+                all_perc: ((d[name] / d3.sum(dice_cnt.map(function(d) {
+                        return d[name];
+                    }))) * 100).toFixed(2)
+            };
+        });
+    });
     return dice_cnt;
 
 }
 
 function updateDiceData() {
     var player
-    if(this.nodeName == 'text')
+    if (this.nodeName == 'text')
```

Comment:
hahaha, this was when I was playing around with ability to do user selects. I kinda hacked it in and forgot to clean it up.  Give me a sec.